demo/index.html

```diff
@@ -180,7 +201,13 @@ <h1>${this.title}</h1>
         console.log('Drawer Closed', event);
         menuOpen = false;
       };
-      const menuSliderElement = html`<ia-menu-slider .menus=${menus} @ItemNavMenuClosed=${closeCB} ?open=${menuOpen}></ia-menu-slider>`;
+      const menuSliderElement = html`
+        <ia-menu-slider
+          .menus=${menus}
+          @ItemNavMenuClosed=${closeCB}
```

Comment:
If I am not wrong, you have changed `ItemNavMenuClosed` with `menuSliderClosed` in `src/ia-menu-slider.js` file?

Reply:
thanks for catching